context/src/junit/bindings.rs

```diff
@@ -80,7 +86,8 @@ impl Into<Report> for BindingsReport {
                 .and_then(|micro_secs| {
                     DateTime::from_timestamp(
                         micro_secs / MICROSECONDS_PER_SECOND,
-                        (micro_secs % MICROSECONDS_PER_SECOND) as u32,
+                        ((micro_secs % MICROSECONDS_PER_SECOND) * NANOSECONDS_PER_MICROSECOND)
+                            as u32,
```

Comment:
Rather than doing the math ourselves, how about using `chrono` to do the work for us?
https://docs.rs/chrono/latest/chrono/struct.TimeDelta.html

That way we don't run into human error, and we don't have to write tests for our own time conversions

Reply:
Totally - thanks for the tip